xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1120,6 +1121,7 @@ core.viewers.attachments.showHistory=View attachment history
 core.viewers.attachments.author=Posted by {0}
 core.viewers.attachments.date=on {0}
 core.viewers.attachments.noAttachments=No attachments for this page
+core.viewers.attachments.upload.comment=Add summary...
```

Comment:
comment or summary?

Reply:
This is confusing, but since I use the field `comment`, and also existing code that explicitly mentions it this way, it's a bit hard to tell which one makes the most sense as a label. I guess this one could be summary, but `core.viewers.attachments.livetable.comment` very clearly references the comment field of an attachment.

Reply:
Might be good to do a quick poll to the devs on the matrix chat for the terminology.

Reply:
I would simply follow the same as what we do on document side for the same need: https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties#L940-L966

Reply:
What happened in the past is that we called it "comment" in the DB and in the code, hence the translation keys with "comments", the apis (e.g. `$xwiki.hasEditComment()`, `isEditCommentFieldHidden`) and configuration keys.  It was implemented in https://jira.xwiki.org/browse/XWIKI-114

Then in https://jira.xwiki.org/browse/XWIKI-5526 it was decided to rename "Comments" in "Summary" in the UI (too bad we can't see http://markmail.org/thread/y4gnmnwfdglpygkj anymore).

So, if we want to do the same, then we should keep the concept of "Comments" in the code and in the APIs/configuration and only use Summary in the UI translation keys.